README.md

```diff
@@ -41,6 +41,9 @@ the OpenShift platform and any custom changes to the the default CR will be
 overwritten by the operator.  For custom tuning, create your own tuned CRs.
 Newly created CRs will be combined with the default CR and custom tuning
 applied to OpenShift nodes based on node/pod labels and profile priorities.
+While in certain situations the support for pod labels can be a convenient
+way of automatically delivering required tuning, this practice is discouraged
+and strongly advised against especially in large-scale clusters.
```

Comment:
Could you add a note on how to enable pod label support in the doc as well?

Reply:
Thank you, will add.